routes/analysis_routes.py

```
@@ -97,139 +99,171 @@
+            
+        if result:
+            return jsonify({'success': True})
+        else:
+            return jsonify({'success': False}), 500
+        
+cohort_get_list = reqparse.RequestParser()
+cohort_get_list.add_argument("info_db_no", type=int, required=True, help="Info DB 번호(필수)")
+cohort_get_list.add_argument("analysis_type", type=str, required=False, help="분석 타입(PCL, SubscriptionType, FavGenre, LastLogin)")
+
+@analysis_ns.route('/cohort/list')
+class CohortAnalysis(Resource):
```

Ruff (0.11.9)

Redefinition of unused `CohortAnalysis` from line 213

(F811)